homework_4/task_2.py

```diff
@@ -0,0 +1,20 @@
+import random
+
+# დავწეროთ პროგრამა, რომელიც მიიღებს მთელ დადებით რიცხვს n
+n = 100 #ეს უნდა იყოს 0-დან 30-მდე
+
+while n > 30 or n < 0:
+    n = int(input("Enter positive number between 0 and 30: "))
+
+# დააგენერიროს n ცალი შემთხვევითი მთელი რიცხვი 0 - 1000 დიაპაზონიდან
+random_number_list = []
```

Comment:
შენი სწორად მუშაობს, უბრალოდ გთავაზობ ლისტის გარეშე ამოხსნას:
from random import randint
n = int(input('Enter a whole number n so that 0 < n < 30: '))

if not 0 < n < 30:
    print('Entered number is not between 0 and 30.')
    exit(1)

max_n = 0
for i in range(n):
    random_number = randint(0, 1000)
    if random_number > max_n:
        max_n = random_number

print(max_n)